lib/processors/bundlers/moduleBundler.js

```diff
@@ -35,8 +35,11 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  *	<li>
  *		<code>require</code>: A 'require' section is transformed into a sequence of jQuery.sap.require calls. The
  *		list will be resolved like an include pattern list in any of the other sections and for each of the resolved
- *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.requireSync' is
+ *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.require' is
```

Comment:
The usage of `jQuery.sap.require` has already been removed. This was the case in the past, but this text does not fit to the current codebase and should be updated.

Reply:
In addition: The `@typedef {object} ModuleBundleDefinitionSection` lists the available properties, and `async` is still missing there.

Reply:
Done